available_images.md

```diff
@@ -0,0 +1,140 @@
+|  			 MXNet 1.5.1 with Elastic Inference 			       |  			 inference 			 |  			 No 			              |  			 CPU 			     |  			 2.7 (py27), 3.6 (py36) 			 | 763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-inference-eia:1.5.1-cpu-py27-ubuntu16.04       |
+|  			 PyTorch 1.3.1 with Elastic Inference 			     |  			 inference 			 |  			 No 			              |  			 CPU 			     |  			 3.6 (py36) 			             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-inference-eia:1.3.1-cpu-py36-ubuntu16.04     |
+
+
+Prior Versions
+==============
+
+| Framework 			                                  |  			 Job Type 			            |  			 Horovod Options 			              |  			 CPU/GPU 			 |  			 Python Version Options 			             |  			 Example URL 			                                                                                     |
+|---------------------------------------------|------------------------|---------------------------------|------------|---------------------------------------|----------------------------------------------------------------------------------------------------|
+|  			 TensorFlow 2.1 			                           |  			 training 			            | No                              |  			 CPU 			     |  			 2.7 (py27), 3.6 (py36) 			             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.1.0-cpu-py27-ubuntu18.04        |
+|  			 TensorFlow 2.1 			                           |  			 training 			            |  			 No 			                           |  			 GPU 			     |  			 2.7 (py27), 3.6 (py36) 			             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.1.0-gpu-py27-cu101-ubuntu18.04  |
+|  			 TensorFlow 1.15.2 			                        |  			 training 			            | Yes                             |  			 CPU 			     |  			 2.7 (py27), 3.6 (py36), 3.7 (py37) 			 | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.2-cpu-py27-ubuntu18.04       |
+|  			 TensorFlow 1.15.2 			                        |  			 inference 			           | No                              |  			 CPU 			     |  			 2.7 (py27), 3.6 (py36) 			             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.2-cpu-py27-ubuntu18.04       |
+|  			 TensorFlow 1.15.2 			                        |  			 training 			            |  			 Yes 			                          |  			 GPU 			     |  			 2.7 (py27), 3.6 (py36), 3.7 (py37) 			 | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.2-gpu-py27-cu100-ubuntu18.04 |
+|  			 TensorFlow 1.15.2 			                        |  			 inference 			           |  			 No 			                           |  			 GPU 			     |  			 2.7 (py27), 3.6 (py36) 			             | 763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.2-gpu-py27-cu100-ubuntu18.04 |
```

Comment:
The inference images here should have image URIs:
`763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:1.15.2-cpu-py27-ubuntu18.04`
and
`763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:1.15.2-gpu-py27-cu100-ubuntu18.04`

Reply:
Fixed